lib/plugins/filter/after_post_render/external_link.js

```diff
@@ -2,6 +2,10 @@
 
 const { isExternalLink } = require('hexo-util');
 let EXTERNAL_LINK_POST_ENABLED = true;
+const rATag = /<a(?:\s+?|\s+?[^<>]+\s+?)?href=["']([^<>"']+)["'][^<>]*>/gi;
+const rTargetAttr = /target=/i;
+const rRelAttr = /rel=/i;
+const rRelStrAttr = /rel=["']([^<>"']*)["']/i;
```

Comment:
- Optimized `rATag` results in 20% less steps during regex matching
- Cache regexp when hexo is loaded